biogrid/apps/biogrid-mvp/src/main.tsx

```diff
@@ -3,4 +3,11 @@ import ReactDOM from 'react-dom';
 
 import App from './app/app';
 
+declare global {
+  interface GlobalFetch {
+    [k: string]: any;
+  }
+}
+window.globalThis = window
```

Comment:
what's the point of this?

Reply:
> what's the point of this?

This solves an error at runtime caused by one of the built files by tsoa.